src/components/dialogs/parameters/sensi/sensitivity-table.tsx

```diff
@@ -68,45 +64,36 @@ const SensitivityTable: FunctionComponent<SensitivityTableProps> = ({
-                                    sx={{
-                                        backgroundColor: column.color,
-                                    }}
-                                >
-                                    <FormattedMessage id={column.label} />
-                                </Box>
-                            </TableCell>
-                        ))}
-                        <TableCell>
-                            <Tooltip
-                                title={intl.formatMessage({
-                                    id: 'AddRows',
-                                })}
+                    {columnsDefinition.map((column: any) => (
+                        <TableCell
+                            key={column.dataKey}
+                            sx={{ width: column.width, textAlign: 'center' }}
+                        >
+                            <Box
+                                sx={{
+                                    justifyContent: 'space-between',
+                                    alignItems: 'center',
+                                    fontWeight: 'normal',
+                                    margin: 1,
+                                }}
                             >
-                                <Box>
-                                    <IconButton
-                                        color="primary"
-                                        onClick={handleAddRowsButton}
-                                    >
-                                        <AddCircleIcon
-                                            sx={{
-                                                color:
-                                                    getLocalStorageTheme() ===
-                                                    DARK_THEME
-                                                        ? 'white'
-                                                        : 'black',
-                                            }}
-                                        />
-                                    </IconButton>
-                                </Box>
-                            </Tooltip>
+                                {column.label}
+                            </Box>
                         </TableCell>
-                    </TableRow>
+                    ))}
+                    <TableCell sx={{ width: '5rem', textAlign: 'center' }}>
+                        <Tooltip
+                            title={intl.formatMessage({
+                                id: 'AddRows',
+                            })}
+                        >
+                            <IconButton onClick={handleAddRowsButton}>
+                                <AddCircleIcon />
+                            </IconButton>
+                        </Tooltip>
+                    </TableCell>
                 </TableHead>
                 <TableBody>
                     {currentRows.map((row: any, index: number) => (
```

Comment:
row have the type: UseFieldArrayReturn from 'react-hook-form/dist/types';

Reply:
done